Source/engine/dx.cpp

```diff
@@ -94,6 +94,7 @@ void dx_init()
 	SDL_ShowWindow(ghMainWnd);
 #endif
 
+	Palette = SDLWrap::AllocPalette();
 	palette_init();
 	CreateBackBuffer();
 	pal_surface_palette_version = 1;

```

Comment:
I guess we don't need this anymore, since `palette_init()` already did it.
```suggestion
```

Reply:
Done. In fact, we no longer use `pal_surface_palette_version` at all, so I've removed it completely.